test/Scientist.Test/Scientist.Test.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFrameworks>net8.0;net48</TargetFrameworks>
```

Comment:
Adding `net6.0` here yields an error:
```text
error : Microsoft.NET.Test.Sdk doesn't support net6.0 and has not been tested with it. Consider upgrading your TargetFramework to net8.0 or later.
You may also set <SuppressTfmSupportBuildErrors>true</SuppressTfmSupportBuildErrors> in the project file to ignore this warning and attempt to run in this unsupported configuration at your own risk.
```

If you do set `SuppressTfmSupportBuildErrors=true` there are other errors.

Reply:
Ah it will be because i have 8 & 9 installed!

| Since .NET 6 is out of support I don't think we need to got out of our way to test it going forward.

I completely agree